packages/blockchain-wallet-v4-frontend/src/scenes/Login/model.tsx

```diff
@@ -3,24 +3,62 @@ import { FormattedMessage } from 'react-intl'
 import { LinkContainer } from 'react-router-bootstrap'
 import styled from 'styled-components'
 
-import { Button, Icon, Link, SpinningLoader, Text, TextGroup } from 'blockchain-info-components'
+import {
+  Banner,
+  Button,
+  Icon,
+  Link,
+  SpinningLoader,
+  Text,
+  TextGroup
+} from 'blockchain-info-components'
 import { FormLabel } from 'components/Form'
+import { Wrapper } from 'components/Public'
 import { LoginFormType, LoginSteps } from 'data/types'
 import { media } from 'services/styles'
 
 export const LOGIN_FORM_NAME = 'login'
```

Comment:
made a comment below about this, but lets remove this and use the one in data.auth.model

Reply:
you could actually just import from data here and then re-export for this model file so you dont have to track down all the other spots and update